components/Search/SuggestionItem/SuggestionItem.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Link from "next/link";
+
+export const SuggestionItem = ({ suggestion }) => (
+  <div>
+    <h4>
+      {suggestion.guide_name} / {suggestion.guide_section}
```

Comment:
I **suggest** adding the name of the manual to the search 

**Cause:** sections can be duplicated in different manuals